CONTRIBUTING.md

```diff
@@ -72,6 +73,13 @@ if needed.
 
 ## Contributing code
 
+### Where to find issues
+
+While the Qiskit community owns several projects in GitHub, we are asking for external contributors to please focus on the [qiskit.org repository](https://github.com/Qiskit/qiskit.org), for now. 
+
+To give our collaborators an idea of where the team needs help, we use the [contributions welcome](https://github.com/Qiskit/qiskit.org/issues?q=is%3Aopen+is%3Aissue+label%3A%22contributions+welcome%22) label – this is appropriate for all contributors. In addition, for those who are relatively newer to the open-source workflow or our codebase, feel free to view issues tagged with the [good first issue](https://github.com/Qiskit/qiskit.org/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label.
+
```

Comment:
Maybe could add a sentence about checking that the issue is not yet assigned top anyone before starting to work on it?

Reply:
Added a small section here

https://github.com/Qiskit/qiskit.org/pull/1921/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R108